server/webapp/WEB-INF/rails.new/app/assets/javascripts/environment_variables.js

```diff
@@ -138,8 +137,12 @@ EnvironmentVariables = function() {
         });
     };
 
-    function createWrapperElement(tag_type) {
-        return jQuery(document.createElement(tag_type));
+    function createWrapperElement(tag_type, template_text) {
+        if(tag_type){
+          return jQuery(document.createElement(tag_type)).html(template_text);
+        } else {
+          return jQuery(template_text);
```

Comment:
JQuery is smart about detecting if the "selector" is actually HTML text and creates an element accordingly. http://jsfiddle.net/r16muadb/1/